README.md

```diff
@@ -45,7 +45,7 @@ A problem daemon is a sub-daemon of node-problem-detector. It monitors a specifi
 kind of node problems and reports them to node-problem-detector.
 
 A problem daemon could be:
-* A tiny daemon designed for dedicated usecase of Kubernetes.
+* A tiny daemon designed for the dedicated Kubernetes use-case.
```

Comment:
I'd prefer:
* A tiny daemon designed for dedicated Kubernetes use-cases.

Reply:
(Thanks, I actually struggled with trying to come up w/ a good way to write this.)